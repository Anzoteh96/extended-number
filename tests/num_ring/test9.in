708743072214208859217069331
-487498403164525840180554138
305184276452750551095780868
