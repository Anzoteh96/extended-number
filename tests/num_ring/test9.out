Enter the number base b:
Enter two numbers n and m:
b is 708743072214208859217069331
n mod b is 221244669049683019036515193
m mod b is 305184276452750551095780868
n + m is 526428945502433570132296061
n - m is 624803464811141327157803656
n * m is 633960345364216034477745511
n ** m is 283807775505413895894826248
